SampleDesignSupportLibrary/app/src/main/res/values/styles.xml

```diff
@@ -17,4 +19,8 @@
         <item name="windowActionModeOverlay">true</item>
     </style>
 
+    <style name="AppDrawerArrowToggle" parent="Base.Widget.AppCompat.DrawerArrowToggle">
+        <item name="color">@color/txt_reverse</item>
```

Comment:
I defined `DrawerArrowToggle` style to change humburger icon color.
http://stackoverflow.com/questions/26439572/how-to-style-the-drawerarrowtoggle-from-android-appcompat-v7-21-library